test/30_debate/clientfunctions.js

```diff
@@ -300,7 +300,7 @@ describe("Debate client functions", () => {
             for (let i = 0; i < DebateConfig.MAX_SUGGESTIONS; ++i) {
                 promises.push(new Promise(resolve => {
                     uniqueClient.emit('suggestQuestion', `Suggestion${i}`, res => {
-                        res.should.equal(true);
+                        res.should.not.equal(false);
```

Comment:
Same comment.

Reply:
suggestQuestion was modified to return false on error and the suggestion id on success (it is a necessary modification for the client app).
res.should.not.equal(false) allows me to verify that there is no error, rather than checking for a particular id.